docs/manuals/userman/gui/viselements/generic/chat.md_template

```diff
@@ -164,7 +164,7 @@ The control's `on_action` callback property is set so that the callback function
 invoked when the "Submit" button is pressed.<br/>
 Here is the code of this function:
 ```python linenums="1"
-users: list[str|Icon] = []
+users: list[str | Icon] = []
```

Comment:
I don't think this syntax will be support for python 3.8 as @FredLL-Avaiga mentioned

Reply:
Correct

Reply:
Fixed